.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,6 +21,7 @@
 ## Checklist before requesting a review
 
 - [ ] I have performed a self-review of my code.
+- [ ] Source code has been verified by linters (`make verify`).
```

Comment:
our PR jobs already confirm/enforce this.  If we want anything in this space i think it would be "PR has passed all pre-merge test jobs".  In other words "don't ask for a review until our CI testing confirms the PR is mergeable, unless you're explicitly asking for an early draft review)  And we can also remove the "unit tests passed locally" checkbox below.

Reply:
ok that makes sense, thank you @bparees

Reply:
@bparees updated as expected